### PR TITLE
Adding single-node OpenShift and three-node OpenShift terms to glossary

### PR DESCRIPTION
**Pull request by aireilly**
Adding _single-node OpenShift_ and _three-node Openshift_ terms to glossary. Merge to main.

### CODE REVIEW

**Comment by bobfuru** [comment]
Grammatically, I'd love to see it as "single-node OpenShift" because it's a compound adjective and ISG suggests:
> use hyphens in compound adjectives to show which words modify another word if it is
necessary to avoid ambiguity
But it's not that big of a deal and not all that ambiguous without the hyphen. 

In the interest of consistency and solidarity, this LGTM and I'd love to see this PR merged!

**Comment by aireilly** [comment]
> Grammatically, I'd love to see it as "single-node OpenShift" because it's a compound adjective and ISG suggests:
> 
> > use hyphens in compound adjectives to show which words modify another word if it is
> 
> necessary to avoid ambiguity
> 
> But it's not that big of a deal and not all that ambiguous without the hyphen. 
> 
> 
> 
> In the interest of consistency and solidarity, this LGTM and I'd love to see this PR merged!

Works for me! Will update next week.

**Comment by aireilly** [comment]
I think we also need an entry for three-node OpenShift compact cluster. Added to PR.

**Comment by aireilly** [comment]
Thanks @bobfuru. @bergerhoffer @kalexand-rh any comments or suggestions? Does anyone else need to give LGTM on this? Someone from the OCP PM side maybe?

**Comment by kalexand-rh** [comment]
@sdodson, will you PTAL at this clarification for the openshif-docs contrib guide and either +1 it or suggest another reviewer?

File: contributing_to_docs/term_glossary.adoc [hunk 1]
```diff
@@ -635,6 +635,24 @@ A service account binds together:
...
+Do not use: Single Node Openshift (SNO).
+
+''''
+=== three-node OpenShift
+
+Usage: three-node OpenShift
+
+Three-node OpenShift is a compact cluster deployment footprint on three nodes for deployments in constrained environments. It provides three control plane nodes that you configure as schedulable for workloads.
+
+Do not use: Three Node Openshift.
```
Comment (sdodson) [inline contributing_to_docs/term_glossary.adoc:654]: Should "compact cluster" or variations of that be discouraged as well in favor of this new term?

Comment (aireilly) [reply contributing_to_docs/term_glossary.adoc:654]: Maybe? I don't mind the phrase "compact cluster" as a synonym for three-node cluster. It's in use widely already.

Comment (aireilly) [reply contributing_to_docs/term_glossary.adoc:654]: @sdodson do you have any more thoughts on this? Is there someone else that should review or comment?

Comment (bobfuru) [reply contributing_to_docs/term_glossary.adoc:654]: Going to merge this, so if there are more thoughts about "compact cluster", we can address in a separate PR.

**Pull request merged**